modules/apps/callbacks/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.21-alpine3.18 as builder
+
+RUN set -eux; apk add --no-cache git libusb-dev linux-headers gcc musl-dev make;
```

Comment:
I guess make isn't necessary anymore? Happy to leave in

Reply:
yeah, good point. Guess it wouldn't hurt to leave it if theres a good change a Makefile might be added.